lib/tasks/rss.rake

```diff
@@ -0,0 +1,42 @@
+namespace :rss do
```

Comment:
This can probably go away if we rely on the initializer, or maybe we can leave it so other people can put their own blog RSS and schedule it with cron...